freqtrade/configuration.py

```diff
@@ -375,22 +376,41 @@ def get_config(self) -> Dict[str, Any]:
 
         return self.config
 
-    def check_exchange(self, config: Dict[str, Any]) -> bool:
+    def check_exchange(self, config: Dict[str, Any], check_for_bad: bool = True) -> bool:
         """
         Check if the exchange name in the config file is supported by Freqtrade
-        :return: True or raised an exception if the exchange if not supported
+        :param check_for_bad: if True, check the exchange against the list of known 'bad'
+                              exchanges
+        :return: False if exchange is 'bad', i.e. is known to work with the bot with
+                 critical issues or does not work at all, crashes, etc. True otherwise.
+                 raises an exception if the exchange if not supported by ccxt
+                 and thus is not known for the Freqtrade at all.
         """
-        exchange = config.get('exchange', {}).get('name').lower()
-        if not is_exchange_supported(exchange):
-
-            exception_msg = f'Exchange "{exchange}" not supported.\n' \
-                            f'The following exchanges are supported: ' \
-                            f'{", ".join(supported_exchanges())}'
+        logger.info("Checking exchange...")
 
-            logger.critical(exception_msg)
+        exchange = config.get('exchange', {}).get('name').lower()
+        if not is_exchange_known(exchange):
             raise OperationalException(
-                exception_msg
+                    f'Exchange "{exchange}" is not supported by ccxt '
+                    f'and not known for the bot.\n'
+                    f'The following exchanges are supported by ccxt: '
+                    f'{", ".join(known_exchanges())}'
             )
 
-        logger.debug('Exchange "%s" supported', exchange)
+        logger.info(f'Exchange "{exchange}" is supported by ccxt and known for the bot.')
```

Comment:
This line and the line 2 below create a double-confirmation, which i don't think is needed.

also "known for the bot" seems oddly worded. I like the other message better :) 

```
2019-06-12 19:31:32,208 - freqtrade.configuration - INFO - Checking exchange...
2019-06-12 19:31:32,208 - freqtrade.configuration - INFO - Exchange "binance" is supported by ccxt and known for the bot.
2019-06-12 19:31:32,208 - freqtrade.configuration - INFO - Exchange "binance" is officially supported by the Freqtrade development team.
```